### PR TITLE
Unit Testing Frameworks task is completed

### PR DESCRIPTION
**Pull request by NadzeyaKiz**
Task is done

### CODE REVIEW

File: Code/TestProject1/CalcNUnitTests.cs [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+        [Test]
+        [TestCase(2, 3, 5, "Sum of positive and positive")]
+        [TestCase(0, 0, 0, "Sum of zero and zero")]
+        [TestCase(0, 5, 5, "Sum of zero and positive")]
+        [TestCase(6, 0, 6, "Sum of positive and zero")]
+        [TestCase(-5, 5, 0, "Sum of negative and positive")]
+        [TestCase(5, -5, 0, "Sum of positive and negative")]
+        [TestCase(-5, -5, -10, "Sum of negative and negative")]
+        public void AddReturnsCorrectSumTest(double a, double b, double expectedSum, string testcaseDescription)
```
Comment (MariaEroshevich) [inline Code/TestProject1/CalcNUnitTests.cs]: No need to make the description as test parameter.

File: Code/TestProject2/CalcXUnitTests.cs [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+            expectedResult = Math.Sqrt(number);
+            double result = calculator.Sqrt(number);
+            Assert.Equal(expectedResult, result);
+        }
+               
+        [Theory]
+        [InlineData(5, true)]
+        [InlineData(-5, false)]
+        [InlineData(0, false)]
+        public void isPositiveReturnsCorrectValueTest(double number, bool expectedValue)
```
Comment (MariaEroshevich) [inline Code/TestProject2/CalcXUnitTests.cs:74]: it is better to split negative and positive verifications to different test methods.

File: Code/TestProject2/CalcXUnitTests.cs [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+        {
+            //Precondition: A double number number to check if it is positive, and the expected boolean expectedValue.
+            //Test: Chek that the isPositive method correctly determines whether the given number is positive.
+            bool actualResult = calculator.isPositive(number);
+            Assert.Equal(expectedValue, actualResult);
+        }
+
+        [Theory]
+        [InlineData(5, false)]
```
Comment (MariaEroshevich) [inline Code/TestProject2/CalcXUnitTests.cs:83]: it is better to split negative and positive verifications to different test methods.

**Review by MariaEroshevich — CHANGES_REQUESTED** [review]

File: Code/TestProject1/CalcNUnitTests.cs [hunk 4]
```diff
@@ -0,0 +1,155 @@
+using CSharpCalculator;
```
Comment (MariaEroshevich) [inline Code/TestProject1/CalcNUnitTests.cs:1]: duplicate

**Review by MariaEroshevich — CHANGES_REQUESTED** [review]

File: Code/TestProject1/CalcNUnitTests.cs (hunk 4 above)
Comment (NadzeyaKiz) [reply Code/TestProject1/CalcNUnitTests.cs:1]: deleted

File: Code/TestProject2/CalcXUnitTests.cs (hunk 2 above)
Comment (NadzeyaKiz) [reply Code/TestProject2/CalcXUnitTests.cs:74]: done

File: Code/TestProject1/CalcNUnitTests.cs (hunk 1 above)
Comment (NadzeyaKiz) [reply Code/TestProject1/CalcNUnitTests.cs]: comment applied

**Review by MariaEroshevich — APPROVED** [review]

**Pull request merged**